### ISSUE TITLE
Error kernel panic not syncing vfs unable to mount root fs on unknown-block(0;0)

### ISSUE THREAD

**Issue by mixu1308** [raspberrypi/noobs#286]
I have tried multiple times to use noobs for multiboot but I keep getting this error:kernel panic not syncing vfs unable to mount root fs on unknown-block(0;0).
I have tried installing openelec and raspbian and both of them worked.
I can't find help anywhere on the internet about this error for NOOBS.
Any help is greatly appreciated.

**Comment by popcornmix** [comment]
I think NOOBS is on the 3.18 kernel, and raspbian/openelec are on 4.1 kernel which has additional sdcard fixes. NOOBS really needs updating to a later kernel.

**Comment by mixu1308** [comment]
So I should just wait?

**Comment by popcornmix** [comment]
Yes. Or try a different sdcard.

**Comment by mixu1308** [comment]
Ok so I tried installing noobs 1.4.2 again just to be sure and I got it to work since last time I was using 1.4.1. I then did sudo halt to shut it down, and now I get the error again :/

**Comment by lurch** [comment]
Yeah - the "1.4" at the start of both "NOOBS v1.4.1" and "NOOBS v1.4.2" indicates that they're both based on the same "core version" of NOOBS, which means the kernel hasn't changed (when the kernel does change, it'll then be NOOBS v1.5.x).
So if it doesn't work on NOOBS v1.4.1 then I'd expect it to also not work on NOOBS v1.4.2

See also https://github.com/raspberrypi/noobs/issues/287#issuecomment-149877958

**Comment by mixu1308** [comment]
But the thing is that it worked once and then it broke and now I get the same error again

**Comment by lurch** [comment]
You were probably just "lucky" that it worked once. From what (little) I understand, this is a subtle timing problem with particular SD cards (with the newer kernel driver being more robust).

As @popcornmix says, your best bet for now is to try a different SD card. http://swag.raspberrypi.org/collections/frontpage/products/noobs-8gb-sd-card is very reliable and has never caused any problems.

**Comment by procount** [comment]
This sounds very similar to https://www.raspberrypi.org/forums/viewtopic.php?f=91&t=123446 where the rootfs could not be found due to an error in cmdline.txt.

**Comment by lurch** [comment]
Similar error-message, but a different cause ;-)
https://github.com/raspberrypi/firmware/issues/397

**Comment by mixu1308** [comment]
Anyway thanks for the info,  is there an ETA? Do you have any recommended micro SD cards that work with the raspberry Pi 2?

**Comment by lurch** [comment]
No ETA yet.
The card I linked to earlier is actually a microSD card in a full-size SD adaptor :-) It works fine on all models of Raspberry Pi, and is in fact the card I use for all my NOOBS testing.

**Comment by mixu1308** [comment]
Just wanted to say that 1.5 solved my issue

**Comment by HOD42** [comment]
I had a similar issue when I swapped a card from a PiZeroW to a regular PiZero. I didn't do a clean shutdown (the system was supposed to be using tempfs for everything). Anyhow, on booting the PiZero it was failing to boot with the "Error kernel panic not syncing vfs unable to mount root fs on unknown-block(0;0)" message. I simply inserted the card in a Windows PC and renamed the /boot/cmdline.txt (effectively removing it). On reboot, it worked fine!
